### PR TITLE
Gazelle: make the default mode 'fix'

### PR DESCRIPTION
**Pull request by pmbethe09**
After this and the go_prefix PR https://github.com/bazelbuild/rules_go/pull/128 in most cases after the initial setup, you can just run 'gazelle' with no arguments.

### CODE REVIEW

File: go/tools/gazelle/gazelle/main.go [hunk 1]
```diff
@@ -32,7 +32,7 @@ import (
 var (
 	goPrefix = flag.String("go_prefix", "", "go_prefix of the target workspace")
 	repoRoot = flag.String("repo_root", "", "path to a directory which corresponds to go_prefix, otherwise gazelle searches for it.")
-	mode     = flag.String("mode", "print", "print, fix or diff")
+	mode     = flag.String("mode", "fix", "print, fix or diff")
```
Comment (damienmg) [inline go/tools/gazelle/gazelle/main.go]: Could you also expand the "print, fix or diff" to explain a bit more what each one does?

I guess fix fix the exiting BUILD file, print output the generated build file and diff shows a diff between the generated and the existing (or is it a diff corresponding to what fix do?)

Comment (pmbethe09) [inline go/tools/gazelle/gazelle/main.go]: yes-ish.
before https://github.com/bazelbuild/rules_go/pull/128
print prints what would be written, fix actually overwrites, diff shows the diff of the overwrite to current.

after https://github.com/bazelbuild/rules_go/pull/128
print prints what was generated, 
diff shows the diff of the overwrite to current.
while fix, will do a "merge" in place.

So I suppose 128 should be fixed to do the merge regardless, and then do one of the three operations.

Comment (pmbethe09) [inline go/tools/gazelle/gazelle/main.go]: Fixed https://github.com/bazelbuild/rules_go/pull/128 to merge in all cases.

Added more info on each mode.

File: go/tools/gazelle/gazelle/main.go [hunk 2]
```diff
@@ -32,7 +32,7 @@ import (
 var (
 	goPrefix = flag.String("go_prefix", "", "go_prefix of the target workspace")
 	repoRoot = flag.String("repo_root", "", "path to a directory which corresponds to go_prefix, otherwise gazelle searches for it.")
-	mode     = flag.String("mode", "fix", "print, fix or diff")
+	mode     = flag.String("mode", "fix", "print: which prints all of the updated BUILD files\n\tfix: which rewrites all of the BUILD files in place\n\tdiff: which computes the rewrite but then just does a diff")
```
Comment (damienmg) [inline go/tools/gazelle/gazelle/main.go]: nit: remove which for all sentence :)

Comment (pmbethe09) [inline go/tools/gazelle/gazelle/main.go]: done

**Pull request merged**